scripts/capture_notimplemented_responses.py

```diff
@@ -191,7 +191,7 @@ def run_script(services: list[str], path: None):
         )
         aggregated_w = csv.DictWriter(
             aggregatefile,
-            fieldnames=["service", "operation", "implemented_count", "full_count", "percentage"],
```

Comment:
q: Why has the operation been removed here? Is it because the aggregation is across all operations of a specific service? What was contained in this field before?

Reply:
i briefly mentioned it in the PR description:

> exclude operation field in the coverage aggregation output (seems like copy-paste-error, it aggregates data for the entire service)

The aggregation was for the entire service (how many operations are implemented, how many operations exist). The `operation` seems to be a left-over or copy-paste from the other `full_w` that we collect before (where we really collect data for each operation). 
We only write to the `aggregated_w` file once for each service, after we looped all operations (for `full_w`) - consequently the operation name was only the last operation of the service and that doesn't make sense :)